### ISSUE TITLE
TypeError: Cannot read property 'getElementsByClassName' of null 

### ISSUE THREAD

**Issue by olskar** [amcolash/MMM-GoogleFit#17]
I see the following in chrome inspector after adding the module:

`TypeError: Cannot read property 'getElementsByClassName' of null
    at moduleNeedsUpdate (main.js:176)
    at main.js:144
    at new Promise (<anonymous>)
    at updateDomWithContent (main.js:137)
    at main.js:119`

**Comment by amcolash** [comment]
I don't believe this is directly specific to my module based on the stack trace. Have you updated the module via `git pull`? Is it still an issue?
